zabbix/files/default/etc/zabbix/zabbix_agentd.conf.jinja

```diff
@@ -127,8 +127,10 @@ LogRemoteCommands={{ settings.get('logremotecommands', defaults.logremotecommand
 # Mandatory: yes, if StartAgents is not explicitly set to 0
 # Default:
 # Server=
-{% if settings.get('server', defaults.get('server', False)) %}
+{% if settings.get('server', defaults.get('server', False)) is list %}
 Server={{ settings.get('server', defaults.server)|join(',') }}
+{% elif settings.get('server', defaults.get('server', False)) %}
```

Comment:
```suggestion

{% if settings.get('server', defaults.get('server', False)) is list -%}
Server={{ settings.get('server', defaults.server)|join(',') }}
{% elif settings.get('server', defaults.get('server', False)) -%}
```
By adding new Jinja line you will actually add empty line to rendered file. This small change will cause recreation of this config file for every user who will update formula. For convenience of other formula user it's better to prevent this when possible. Please check is your changes will trigger changes in the config file while there is no changes in pillars.
My suggested change probably can fix this small issue but I have not tested them. For more info, please check [Jinja Whitespace Control](https://jinja.palletsprojects.com/en/2.11.x/templates/#whitespace-control)

Reply:
when I'm testing with master and my branch, i don't get any difference (when using list syntax), if i apply your suggestion, i do get the line removed, though.
```
----------
          ID: /etc/zabbix/zabbix_agent2.conf
    Function: file.managed
      Result: True
     Comment: File /etc/zabbix/zabbix_agent2.conf updated
     Started: 02:34:00.157233
    Duration: 1180.621 ms
     Changes:   
              ----------
              diff:
                  --- 
                  +++ 
                  @@ -96,7 +96,6 @@
                   # Mandatory: yes, if StartAgents is not explicitly set to 0
                   # Default:
                   # Server=
                  -
                   Server=saltminion2.lan
                   
                   ### Option: ListenPort
                  @@ -395,3 +394,4 @@
                   # Default:
                   # TLSPSKFile=
```


just to rule out some differences in our setup, on master, if server is set, do you have
```
# Mandatory: yes, if StartAgents is not explicitly set to 0
# Default:
# Server=

Server=saltminion2.lan
```

or 

```
# Mandatory: yes, if StartAgents is not explicitly set to 0
# Default:
# Server=
Server=saltminion2.lan
```
?
i have the variant with the empty line betweern the comment block and config line, and  i get it on other config lines too, so it makes sense to keep it there for consitency

Reply:
Well, I tested your changes it looks like `elif` does not add extra line in this case. Jinja renderer in my head is far from perfect... My apologies, it' seems no changes required in this part of code.